jupyter_console/_version.py

```diff
@@ -1,2 +1,10 @@
-version_info = (5, 0, 0, 'dev')
-__version__ = '.'.join(map(str, version_info))
+""" For beta/alpha/rc releases, the version number for a beta is X.Y.ZbN 
+**without dots between the last 'micro' number and b**. N is the number of
+the beta released i.e. 1, 2, 3 ...
+
+See PEP 440 https://www.python.org/dev/peps/pep-0440/
+"""
+
+version_info = (5, 0, 0, '.dev')
```

Comment:
Yeah, I'm thinking of having a 

```
if ('dev', 'rc', or `b`) in __version__ :
    import pep440
    pep440.assert_canonical(__version__)
```

But last time I tried @minrk and @takluyver thought it was overkill.